relay-server/src/services/store.rs

```diff
@@ -594,35 +558,48 @@ impl StoreService {
         event_id: EventId,
         project_id: ProjectId,
         item: &Item,
-    ) -> Result<ChunkedAttachment, StoreError> {
+        send_individual_attachments: bool,
+    ) -> Result<Option<ChunkedAttachment>, StoreError> {
         let id = Uuid::new_v4().to_string();
 
         let mut chunk_index = 0;
         let mut offset = 0;
```

Comment:
nit: `chunk_index` is used later on, but `offset` could move into the `else` branch, right?